Source/Projections/Store/IProjectionOf.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Dolittle.SDK.Projections.Store;
+
+/// <summary>
+/// Defines a system that knows about a projection.
+/// </summary>
+/// <typeparam name="TReadModel">The <see cref="Type" /> of the projection read model.</typeparam>
+public interface IProjectionOf<TReadModel>
+    where TReadModel : class, new()
+{
+    /// <summary>
+    /// Gets the <see cref="ScopedProjectionId"/> identifier.
+    /// </summary>
+    public ScopedProjectionId Identifier { get; }
```

Comment:
1) No need for public here.
2) `ScopedProjectionId` feels a bit internal to the SDK. Maybe we should split it up into the `ProjectionId` and `ScopeId` in two properties?